kratos/tests/test_variable_utils.py

```diff
@@ -4,6 +4,7 @@
 
 import math
 import os
+import numpy as np

```

Comment:
If you use numpy, please add the decorator to skip the tests. It is used in other tests depending on numpy

Reply:
no, numpy is now accepted